src/main/resources/application.yaml

```diff
@@ -206,6 +209,16 @@ notification:
     smsId: 04e778fd-71a8-4c27-b423-aa7485a81a9c
   struckOut:
     letterId: 7383b637-d76e-4c9f-8af9-3e3573168011
+  interlocValidAppeal:
+    appellant:
+      emailId: 01293b93-b23e-40a3-ad78-2c6cd01cd21c
+      letterId: 91143b85-dd9d-430c-ba23-e42ec90f44f8
+    appointee:
+      emailId: 01293b93-b23e-40a3-ad78-2c6cd01cd21c
+      letterId: 91143b85-dd9d-430c-ba23-e42ec90f44f8
```

Comment:
Shouldn't these letters only be sent when the 'caseUpdated' event is received? Not the interlocValidAppeal event

Reply:
Josh is check the interlocValidAppeal templates now, I will follow up when he feeds b